admin-api/admin-api-specification-v1.0.md

```diff
@@ -1,4 +1,906 @@
+
+> **NOTE:** The `DELETE` method is not supported.
+
+## HTTP response status codes
+
+[Table "HTTP response status codes"](#table-http-response-status-codes) lists the HTTP response status codes that the API supports:
+
+|Status Code|Reason|Description|
+|---|---|---|
+|**200**|OK|Standard response for a successful `GET`, `PUT`, or `POST` operation. The response will contain an entity corresponding to the requested resource.|
+|**201**|Created|The `POST` request has been fulfilled, resulting in the creation of a new resource. The response will not contain an entity describing or containing the result of the action.|
+|**202**|Accepted|The request has been accepted for processing, but the processing has not been completed.|
+|**400**|Bad Request|The server could not understand the request due to invalid syntax.|
+|**401**|Unauthorized|The request requires authentication in order to be processed.|
+|**403**|Forbidden|The request was denied and will be denied in the future.|
+|**404**|Not Found|The requested resource is not available at the moment.|
+|**405**|Method Not Allowed|An unsupported HTTP method for the request was used.|
+|**406**|Not Acceptable|The requested resource is capable of generating only content not acceptable according to the Accept headers sent in the request.|
+|**500**|Internal Server Error|A generic error message, given when an unexpected condition was encountered and no more specific message is suitable.|
+|**501**|Not Implemented|The server does not support the requested service. The client should not retry.|
+|**503**|Service Unavailable|The server is currently unavailable to accept any new service requests. This should be a temporary state, and the client should retry within a reasonable time frame.|
+
+# API services
+
+This section introduces and details all services that the API supports for each resource and HTTP method.
+
+## High-level API services
+
+On a high level, the API can be used to perform the following actions:
+
+- View, create, update participant-related details, such as limit (Net Debit Cap), position, or endpoints configured.
+  - Use the services provided by the API resource **`/participants`**.
+- View, create, update settlement-model-related details, such as granularity, delay, liquidity check, and so on.
+  - Use the services provided by the API resource **`/settlementModels`**.
+- View transaction details for a particular transfer.
+  - Use the services provided by the API resource **`/transactions`**.
+
+### Supported API services
+
+[Table "Supported API services"](#table-supported-api-services) includes high-level descriptions of the services that the API provides. For more detailed information, see the sections that follow.
+
+| URI  |  HTTP method `GET`| HTTP method `PUT` | HTTP method `POST` | HTTP method `DELETE` |

```

Comment:
You've already said that `DELETE` is not supported above... let's remove it from this table and make it easier to read.

Reply:
I would like to leave this as is if it's OK with you. Readers don't necessarily read in a linear fashion. Users often scan-read for bits of information they are interested in. A reader who decided to give their attention to this table may not have read the section titled "HTTP methods" (which is where we first stated that `DELETE` is not supported).

Reply:
Yep, 100% agree.